sbin/ncbi-parse-geneinfo

```diff
@@ -30,8 +30,6 @@ if __name__ == "__main__":
     giw = GeneInfoWriter(sys.stdout)
 
     for rec in gi_in:
-        if rec["symbol_from_nomenclature_authority"] == "-":
```

Comment:
Removing this here instead of creating another ticket. We do not want to skip gene imports that do not have a symbol from HGNC.